kubernetes/kube-state-metrics-role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  namespace: kube-system
+  name: pod-nanny
+rules:
+- apiGroups: [""]
+  resources:
+  - pods
+  resourceNames: ["kube-state-metrics"]  
+  verbs: ["get"]
+- apiGroups: ["extensions"]
+  resources:
+  - deployments
+  resourceNames: ["kube-state-metrics"]
+  verbs: ["get", "update"]
```

Comment:
the nanny updates is using https://github.com/kubernetes/autoscaler/blob/master/addon-resizer/nanny/kubernetes_client.go#L69